features/support/files_helpers.rb

```diff
@@ -23,6 +23,32 @@ def files_in_branches
 end
 
 
+def create_uncommitted_file options = {}
+  options.reverse_merge! default_uncommitted_file_attributes
+  IO.write options[:file_name], options[:file_content]
+  options[:file_name]
+end
+
+
+def default_uncommitted_file_attributes
```

Comment:
Good idea to dry this up by separating these atttributes into their own place! You could also make this a constant.

```ruby
DEFAULT_UNCOMMITTED_FILE_ATTRIBUTES = {
  file_name: 'uncommitted_file',
  file_content: 'default uncommitted content'
}.freeze
```